cmd/bosun/expr/funcs.go

```diff
@@ -344,6 +345,14 @@ func DropNA(e *State, T miniprofiler.Timer, series *Results) (*Results, error) {
 	}
 	return DropValues(e, T, series, 0, dropFunction)
 }
+func printGraphiteError(err error) {
```

Comment:
This function can be removed since there's only one place this kind of error can be produced, so just inline it.